pkg/inspector/inspector.go

```diff
@@ -47,6 +50,7 @@ func (s *Session) close() {
 	s.once.Do(func() {
 		s.onClose()
 		close(s.C)
+		measure.InspectorsGauge.WithValues(s.componentID).Dec()
```

Comment:
If we move this call into `Session.onClose` (line 128) then we wouldn't have to store `componentID` in `Session`.